editor/src/templates/index.html

```diff
@@ -53,85 +76,48 @@
-                <div class="text stripes2 w100 h14 br5"></div>
-              </div>
-            </div>
-            <div class="border2 innerShell flex flex-row items-stretch flex-grow h32">
-              <div class="code border2r flex-grow items-stretch flex flex-column">
-                <div class="codeContent stripes4 flex-grow-1 pa16"></div>
-                <div class="codeTabs flex-grow-0 border2t h32 flex flex-row">
-                  <div class="problems tab border2r flex flex-row items-center pa8 w120">
-                    <div class="text stripes2 w100 h14 br5"></div>
-                  </div>
-                  <div class="problems tab border2r flex flex-row items-center pa8 w120">
-                    <div class="text stripes2 w100 h14 br5"></div>
-                  </div>
-                </div>
-              </div>
-              <div class="canvas flex-grow-2 relative">
-                <div
-                  id="loading-message"
-                  class="absolute"
-                  style="
-                    bottom: 30px;
-                    right: 30px;
-                    font-size: 90px;
-                    font-family: Akkurat, Garamond;
-                    -webkit-text-stroke: var(--stroke1);
-                    -webkit-text-stroke-width: 1px;
-                    color: transparent;
-                  "
-                >
-                  Utopia
-                </div>
-                <div
-                  class="the-app border3 absolute overflow-hidden br5"
-                  style="left: 50px; top: 50px; height: 812px; width: 375px;"
-                >
-                  <div
-                    class="loadingAnimation stripes1"
-                    style="width: 1575px; height: 812px;"
-                  ></div>
-                </div>
-              </div>
-              <div class="border2l w240 pt8">
-                <div class="h24 border2b"></div>
-                <div style="height: 25%;" class="icons border2b">
-                  <div class="flex flex-row items-center h36">
-                    <span class="subheader"></span>
-                  </div>
-                </div>
-                <div style="height: 25%;" class="icons border2b">
-                  <div class="pl8 pr8 flex flex-row items-center h36">
-                    <span class="subheader"></span>
-                  </div>
-                </div>
-              </div>
-            </div>
+
+        <div
+          style="
+            position: fixed;
+            left: 0px;
+            top: 0px;
+            bottom: 0px;
+            right: 0px;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+            border: 1px solid beige;
+          "
+        >
+          <img src='/editor/pyramid_fullsize@2x.jpg?hash=%UTOPIA_SHA%'' height='128px' alt='Utopia
+          Logo '/>
```

Comment:
Does the `alt` being split over two lines cause any issues? I'm guessing this was caused by prettier doing its thing

Reply:
Huh, that does look like prettier. It was rendering ok FWIW